docs/introduction.rst

```diff
@@ -50,18 +50,30 @@ Below is a depiction of how the system works in regards to Near-real-time (NRT)
 Build Server and Client
 ---------------------------
 
-In the home directory.
+In the home directory is a Dockerfile that will build a base image that can be used for the client and server.
+This Dockerfile is based off of a Java-14 image, installs the distribution via gradle, and can be built like this:
 
 .. code-block::
-  ./gradlew clean installDist test
-
-Note: This code has been tested on *Java14*.
+  shell% docker build --tag nrtsearch .
 
 Run gRPC Server
 ---------------------------
 
+The server can be run via the base image created in the step above. 
+.. code-block::
+  shell% docker run -d --network host nrtsearch /user/app/build/install/nrtsearch/bin/lucene-server
+
+Run gRPC Slient
```

Comment:
Should this be client?

Reply:
This should be fixed now.  Closing conversation.